### PR TITLE
Sw 5276 adding firm

### PR DESCRIPTION
**Pull request by Nicholassully**

### CODE REVIEW

File: cypress/integration/add_firm.spec.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+                cy.get("li").eq(3).should("contain", "The county must be 255 characters or fewer");
+                cy.get("li").eq(4).should("contain", "The email must be 255 characters or fewer");
+                cy.get("li").eq(5).should("contain", "The firm name is required and can't be empty");
+                cy.get("li").eq(6).should("contain", "The telephone number must be 255 characters or fewer");
+                cy.get("li").eq(7).should("contain", "The postcode must be 255 characters or fewer");
+                cy.get("li").eq(8).should("contain", "The town or city must be 255 characters or fewer");
+            })
+        });
+    })
+});
```
Comment (gurps-singh) [inline cypress/integration/add_firm.spec.js:29]: Should there be an additional test testing you can successfully add a firm?

**Review by gurps-singh — CHANGES_REQUESTED** [review]

File: cypress/integration/add_firm.spec.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+                cy.get("li").eq(2).should("contain", "Address line 3 must be 255 characters or fewer");
+                cy.get("li").eq(3).should("contain", "The county must be 255 characters or fewer");
+                cy.get("li").eq(4).should("contain", "The email must be 255 characters or fewer");
+                cy.get("li").eq(5).should("contain", "The firm name is required and can't be empty");
+                cy.get("li").eq(6).should("contain", "The telephone number must be 255 characters or fewer");
+                cy.get("li").eq(7).should("contain", "The postcode must be 255 characters or fewer");
+                cy.get("li").eq(8).should("contain", "The town or city must be 255 characters or fewer");
+            })
+        });
+    })
```
Comment (denialanderror) [inline cypress/integration/add_firm.spec.js:28]: Can we have a success test as well?

**Review by denialanderror — CHANGES_REQUESTED** [review]

File: cypress/integration/add_firm.spec.js (hunk 1 above)
Comment (Nicholassully) [reply cypress/integration/add_firm.spec.js:36]: done

File: cypress/integration/add_firm.spec.js (hunk 2 above)
Comment (Nicholassully) [reply cypress/integration/add_firm.spec.js:35]: done

**Review by gurps-singh — DISMISSED** [review]

**Review by denialanderror — APPROVED** [review]

**Pull request closed without merging**